BGRSCRPT/iocBoot/iocBGRSCRPT-IOC-01/runIOC.bat

```diff
@@ -2,12 +2,28 @@
 setlocal
 set MYDIRBLOCK=%~dp0
 call C:\Instrument\Apps\EPICS\config_env_base.bat
-
+call dllPath.bat
 %HIDEWINDOW% h
 
 set EPICS_CAS_INTF_ADDR_LIST=127.0.0.1
 set EPICS_CAS_BEACON_ADDR_LIST=127.255.255.255
 
 set PYTHONUNBUFFERED=TRUE
 
-%PYTHON3W% %ICPINSTSCRIPTROOT%/background_script.py
+set "GETMACRO=C:\Instrument\Apps\EPICS\support\icpconfig\master\bin\%EPICS_HOST_ARCH%\icpconfigGetMacro.exe"
```

Comment:
having run config_env_base you can use `%EPICS_KIT_ROOT%` instead of `C:\Instrument\Apps\EPICS`